scripts/broker-ci/mediawiki123.yaml

```diff
@@ -4,8 +4,8 @@ metadata:
   name: mediawiki
   namespace: default
 spec:
-  externalClusterServiceClassName: dh-mediawiki123-apb
-  externalClusterServicePlanName: default
+  clusterServiceClassExternalName: dh-mediawiki123-apb
+  clusterServicePlanExternalName: default
```

Comment:
We should not be leaking service-catalog names into our apbs. It shouldn't matter what these are called as long as they get translated on the broker side. IMO. We should define what we want them to be and use that.